### PR TITLE
[infrastructure] change scopes where necessary

### PR DESCRIPTION
**Pull request by J-N-K**
Cleanup leftovers from scope-changing

Signed-off-by: Jan N. Klug <jan.n.klug@rub.de>

### CODE REVIEW

File: itests/org.openhab.binding.mqtt.homie.tests/src/main/java/org/openhab/binding/mqtt/HomieImplementationTest.java [hunk 1]
```diff
@@ -147,7 +147,7 @@ public void setUp() throws InterruptedException, ExecutionException, TimeoutExce
         futures.add(embeddedConnection.publish(propertyTestTopic + "/$datatype", "boolean".getBytes()));
 
         registeredTopics = futures.size();
-        CompletableFuture.allOf(futures.toArray(new CompletableFuture[0])).get(1000, TimeUnit.MILLISECONDS);
+        CompletableFuture.allOf(futures.toArray(new CompletableFuture[0])).get(1500, TimeUnit.MILLISECONDS);
```
Comment (kaikreuzer) [inline itests/org.openhab.binding.mqtt.homie.tests/src/main/java/org/openhab/binding/mqtt/HomieImplementationTest.java]: This change does not seem to be related to this PR - if you want to change this code, please do it in a separate PR for better traceability.

File: itests/org.openhab.binding.mqtt.homie.tests/src/main/java/org/openhab/binding/mqtt/HomieImplementationTest.java [hunk 2]
```diff
@@ -156,7 +156,7 @@ public void setUp() throws InterruptedException, ExecutionException, TimeoutExce
     public void tearDown() throws InterruptedException, ExecutionException, TimeoutException {
         if (connection != null) {
             connection.removeConnectionObserver(failIfChange);
-            connection.stop().get(500, TimeUnit.MILLISECONDS);
+            connection.stop().get(1000, TimeUnit.MILLISECONDS);
```
Comment (kaikreuzer) [inline itests/org.openhab.binding.mqtt.homie.tests/src/main/java/org/openhab/binding/mqtt/HomieImplementationTest.java]: This change does not seem to be related to this PR - if you want to change this code, please do it in a separate PR for better traceability.

File: itests/org.openhab.binding.mqtt.homie.tests/src/main/java/org/openhab/binding/mqtt/HomieImplementationTest.java (hunk 1 above)
Comment (J-N-K) [reply itests/org.openhab.binding.mqtt.homie.tests/src/main/java/org/openhab/binding/mqtt/HomieImplementationTest.java]: I can remove that. However, it doesn't compile without that for me.

**Comment by kaikreuzer** [comment]
Seems this needs to be rebased now, sorry...
